next/SparseImplicitQuadtree/tileset.json

```diff
@@ -0,0 +1,29 @@
+{
+  "extensions" : {
+    "3DTILES_content_gltf" : { }
+  },
+  "asset" : {
+    "version" : "1.0"
+  },
+  "geometricError" : 1024.0,
+  "root" : {
+    "extensions" : {
+      "3DTILES_implicit_tiling" : {
+        "subdivisionScheme" : "QUADTREE",
+        "subtreeLevels" : 3,
+        "maximumLevel" : 5,
+        "subtrees" : {
+          "uri" : "subtrees/{level}.{x}.{y}.subtree"
+        }
+      }
+    },
+    "boundingVolume" : {
+      "box" : [ 0.5, 0.5, 0.00625, 0.5, 0.0, 0.0, 0.0, 0.5, 0.0, 0.0, 0.0, 0.00625 ]
+    },
+    "geometricError" : 32.0,
+    "refine" : "ADD",
+    "content" : {
+      "uri" : "content/content_{level}__{x}_{y}.b3dm"
```

Comment:
Just checking, is the double underscore intentional?

Reply:
It was intended to more easily separate the level from the x/y coordinates. If desired, I can remove it - or use a different naming pattern altogether - I hesitated and considered to use the same as for the subtrees, i.e.. just `level.x.y.b3dm` ...

Reply:
Only a slight preference for `{level}_{x}_{y}` from me but no strong reason to back it up. @ptrgags do you have an opinion here?